### PR TITLE
add sdl_ttf/2.0.15

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

I don't know if it was a good idea to rename `sdl2` recipe to `sdl`, but I did the same for this one to be compliant.

Adapted from https://github.com/bincrafters/community/pull/1363

This recipe will need a new build of packages when linux support will be added to sdl recipe.

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
Convert to draft because I'm reluctant to name this recipe sdl_ttf instead of sdl2_ttf: https://github.com/conan-io/conan-center-index/pull/5466#issuecomment-863222923

**Review by SSE4 — CHANGES_REQUESTED** [review]
please rename package to `sdl2_ttf`.

File: recipes/sdl_ttf/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            ttf_path = os.path.join(self.source_folder, "OpenSans-Bold.ttf")
```
Comment (madebr) [inline recipes/sdl_ttf/all/test_package/conanfile.py:17]: Perhaps, we should add the LICENSE.txt file from the font archive?
See https://www.1001fonts.com/open-sans-font.html#license

**Comment by SpaceIm** [comment]
@SSE4 waiting https://github.com/conan-io/conan-center-index/pull/5944

File: recipes/sdl_ttf/all/test_package/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/sdl_ttf/all/test_package/conanfile.py:17]: We have this file in several recipes, maybe this license could be added in a shared folder?

**Comment by SpaceIm** [comment]
> please rename package to `sdl2_ttf`.

Following discussions in #5944, I guess that we should keep `sdl_ttf`.

File: recipes/sdl_ttf/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        self.requires("freetype/2.10.4",)
```
Comment (madebr) [inline recipes/sdl_ttf/all/conanfile.py:49]:
```suggestion
        self.requires("freetype/2.10.4")
```

**Review by madebr — DISMISSED** [review]

**Review by madebr — APPROVED** [review]

File: recipes/sdl_ttf/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class SdlttfConan(ConanFile):
+    name = "sdl_ttf"
+    description = "A TrueType font library for SDL"
+    license = "ZLIB"
```
Comment (uilianries) [inline recipes/sdl_ttf/all/conanfile.py:11]:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib

**Review by uilianries — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**